src/main/java/com/google/graphgeckos/dashboard/DashboardApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.google.graphgeckos.dashboard;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
```

Comment:
[Alphabetize imports](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing). Swap ...boot.SpringApplication and ...boot.autoconfigure...